tests/end-to-end/regression/5574.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+https://github.com/sebastianbergmann/phpunit/issues/5574
+--FILE--
+<?php declare(strict_types=1);
+$_SERVER['argv'][] = '--do-not-cache-result';
+$_SERVER['argv'][] = '--no-configuration';
+$_SERVER['argv'][] = __DIR__ . '/5574/Issue5574Test.php';
+
+require_once __DIR__ . '/../../bootstrap.php';
+(new PHPUnit\TextUI\Application)->run($_SERVER['argv']);
+--EXPECTF--
+PHPUnit %s by Sebastian Bergmann and contributors.
+
+Runtime: %s
+
+E                                                                   1 / 1 (100%)
+
+Time: %s, Memory: %s MB
+
+There was 1 error:
+
+1) PHPUnit\TestFixture\Issue5574\Issue5574Test::testOne
+Exception: My exception
+
+%sIssue5574Test.php:14
```

Comment:
unit test like in https://github.com/sebastianbergmann/phpunit/issues/5574 issue description would be good too

Reply:
> unit test like in https://github.com/sebastianbergmann/phpunit/issues/5574 issue description would be good too

I do not understanding what you are suggesting. Would you please re-state it.

Reply:
I suggest to keep the E2E test and to add pure unit test like in issue description with `self::assertSame` asserting the whole redacted stack trace. The reason is the numbers in the E2E tests can be replaced by `%d` by some mass edit someday (unintentionally) and also because unit tests are easier to debug.